.github/ISSUE_TEMPLATE/⌨-accessibility-issue.md

```diff
@@ -2,8 +2,7 @@
 name: "⌨ Accessibility Issue"
 about: An accessibility related issue template.
 title: "⌨ Accessibility Issue"
-labels: 
-  - accessibility
+labels: accessibility, component, epic
```

Comment:
What was the reasoning behind using the `epic` label here?

Reply:
Actually, it shouldn't be 🤔 Something wonky happened with Github's templating UI in both these PRs (deleting a previous template in the other PR, adding labels from one template to another in this PR), so maybe we retain template PRs for instances like this.

Reply:
🤷‍♀️ That is odd, I have never used this feature tbh so I have no clue 😛